### PR TITLE
Changed target_pose for "Planning Around Objects" tutorial

### LINKED ISSUE #762: "Planning Around Objects" tutorial fails to generate motion plan
### Description

The current tutorial given on the website needs to be updated with the [new target_pose](https://github.com/ros-planning/moveit2_tutorials/blob/dd73bd2a7b4b5981c86b7f5dd4366d810358d078/doc/tutorials/planning_around_objects/hello_moveit.cpp#L51) mentioned in the example code. The [current target_pose](https://github.com/ros-planning/moveit2_tutorials/blob/dd73bd2a7b4b5981c86b7f5dd4366d810358d078/doc/tutorials/planning_around_objects/planning_around_objects.rst?plain=1#L32) fails to generate a plan.

While we're at it, this [link](https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/tutorials/planning_around_objects/hello_moveit_kinova.cpp) also needs to be updated on the same tutorial.

### Your environment
* ROS Distro: Rolling
* OS Version: Ubuntu
* Source Build
* Commit dd73bd2a7b4b5981c86b7f5dd4366d810358d078

### Steps to reproduce
Follow along with the tutorial on [this](https://moveit.picknik.ai/main/doc/tutorials/planning_around_objects/planning_around_objects.html) page.

### Expected behaviour
Motion plan cannot be generated based on the given target coordinates.

### Backtrace or Console output

```console
[INFO] [1694106763.233801081] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.240623 seconds
[INFO] [1694106763.233858155] [moveit_robot_model.robot_model]: Loading robot model 'gen3'...
[INFO] [1694106763.233868427] [moveit_robot_model.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[WARN] [1694106763.328922587] [moveit_ros.robot_model_loader]: No kinematics plugins defined. Fill and load kinematics.yaml!
[INFO] [1694106763.343952837] [move_group_interface]: Ready to take commands for planning group manipulator.
[INFO] [1694106763.344339546] [hello_moveit.remote_control]: RemoteControl Ready.
[INFO] [1694106763.350426312] [hello_moveit.remote_control]: Waiting to continue: Press 'Next' in the RvizVisualToolsGui window to plan
[INFO] [1694106766.937359662] [hello_moveit.remote_control]: ... continuing
[INFO] [1694106766.938622140] [move_group_interface]: MoveGroup action client/server ready
[INFO] [1694106766.938941115] [move_group_interface]: Planning request accepted
[INFO] [1694106772.127333221] [move_group_interface]: Planning request aborted
[ERROR] [1694106772.127855455] [move_group_interface]: MoveGroupInterface::plan() failed or timeout reached
[ERROR] [1694106772.127929110] [hello_moveit]: Planning failed!
```

### PR DESCRIPTION
**Pull request by Shobuj-Paul**
### Description

Fixes #762 

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/tutorials/planning_around_objects/planning_around_objects.rst [hunk 1]
```diff
@@ -31,10 +31,12 @@ First, update the target pose with the following change to make the robot plan t
...
-      msg.orientation.w = 1.0;
-      msg.position.x = 0.28;
-      msg.position.y = 0.4;  // <---- This value was changed
-      msg.position.z = 0.5;
+      msg.orientation.y = 0.8;
+      msg.orientation.w = 0.6;
+      msg.position.x = 0.1;
+      msg.position.y = 0.4;
+      msg.position.z = 0.4;
+      msg.orientation.w = 1.0; // All values are changed
```
Comment (sea-bass) [inline doc/tutorials/planning_around_objects/planning_around_objects.rst:39]: You have 2 lines setting `msg.orientation.w` -- I think there's only supposed to be 1?

Comment (Shobuj-Paul) [reply doc/tutorials/planning_around_objects/planning_around_objects.rst:39]: Oh yeah, my bad
I'll make the change

Comment (Shobuj-Paul) [reply doc/tutorials/planning_around_objects/planning_around_objects.rst]: Done!

File: doc/tutorials/planning_around_objects/planning_around_objects.rst [hunk 2]
```diff
@@ -31,10 +31,11 @@ First, update the target pose with the following change to make the robot plan t
...
-      msg.position.x = 0.28;
-      msg.position.y = 0.4;  // <---- This value was changed
-      msg.position.z = 0.5;
+      msg.orientation.y = 0.8;
+      msg.orientation.w = 0.6;
+      msg.position.x = 0.1;
+      msg.position.y = 0.4;
+      msg.position.z = 0.4; // All values are changed
       return msg;
     }();
```
Comment (sea-bass) [inline doc/tutorials/planning_around_objects/planning_around_objects.rst:40]: maybe just grab exactly what is in the C++ file?

```suggestion
  // Set a target Pose with updated values !!!
  auto const target_pose = [] {
    geometry_msgs::msg::Pose msg;
    msg.orientation.y = 0.8;
    msg.orientation.w = 0.6;
    msg.position.x = 0.1;
    msg.position.y = 0.4;
    msg.position.z = 0.4;
    return msg;
  }();

**Review by sea-bass — COMMENTED** [review]
I think you renamed the image file from `planning_around_object.png` to `planning_around_objects.png` so the htmlproofer is failing in CI.

Either revert the image name or update the link in the .rst file

**Review by sea-bass — APPROVED** [review]

**Pull request merged**